README.md

```diff
@@ -49,3 +49,26 @@ connector, as long as the user has proper access to those tables.
 
 Creating a Destination connector will fail if the table does not exist or if the user does not have permission to work
 with the specified table.
+
+## Source
+
+The ClickHouse Source allows you to move data from the ClickHouse table to any Conduit Destination. When the connector
+starts, all the data in the table is transferred to the Destination. After all the data is transferred, the connector
+will transfer the newly added data.
+
```

Comment:
Maybe lets just add a short info about snapshot and cdc iterators?

Reply:
We do not have these iterators in their usual sense. 
So I tried to describe the Source logic without dividing it into iterators.

Reply:
But I don't see this description

Reply:
It is below :)
But I got it.
I'll try to add more information.

Reply:
As I remember you use snapshot max value logic we can add few words about it

Reply:
I've added the info about the iterator.